src/test/java/core/basesyntax/dao/DataWriterTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.exceptions.NullFileNameException;
+import core.basesyntax.model.Fruit;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataWriterTest {
+    private static DataWriter dataWriter;
+    private String dataOutput;
+
+    @BeforeClass
+    public static void beforeClass() {
+        dataWriter = new FruitFileWriter();
```

Comment:
why do we have file reader with initializing file name in constructor and file writer with no fileName constructor initializing?